src/screens/FeedScreen/index.js

```diff
@@ -118,10 +121,6 @@ const FeedScreen = (props) => {
     props.navigation.navigate('DomainScreen', param);
   };
 
-  const onEndReach = () => {
-    getDataFeedsHandle(postOffset, false, nextTargetFeed);
-  };
-
   const refreshMoreText = (index, haveSeeMore) => {
     setUpdateIndex(index);
     if (haveSeeMore) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` function has been removed. This change seems to be part of the new approach to fetch more feeds on scroll end rather than when reaching the end of the list. Just ensure that this doesn't affect other parts of your application where `onEndReach` might have been used.
```diff
-  const onEndReach = () => {
-    getDataFeedsHandle(postOffset, false, nextTargetFeed);
-  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->